code/game/objects/structures/crates_lockers/crates/secure.dm

```diff
@@ -19,8 +19,7 @@
 	if(user)
 		to_chat(user, span_danger("The crate's anti-tamper system activates!"))
 		log_bomber(user, "has detonated a", src)
-	for(var/obj/loot in src)
-		SSexplosions.high_mov_atom += loot
+	dump_contents()
```

Comment:
This subtype is not used only for the abandoned crates, it's also used for secure cargo crates ?

Reply:
You're right, but the pizza crate is the only other one that has this, and that could also be abused for the same thing as abandoned crate. (Unlock, trap someone in it, lock crate, throw stuff at crate until it explodes and qdels them)

Reply:
The whole feature should probably be moved to to abandoned crates only.